resources/views/components/cart.blade.php

```diff
@@ -0,0 +1,18 @@
+<div {{ $attributes }}>
+    <table>
+        <thead>
+            <tr>
+                <th>Description</th>
+                <th>Qty</th>
+                <th>Each</th>
+                <th>Discount</th>
+                <th>Subtotal</th>
+            </tr>
+        </thead>
+        <tbody>
+        @foreach ($cart->getItems() as $item)
+            <x-dynamic-component :component="$item->getViewComponent() ?? 'tipoff-cart-item'" :cart-item="$item" :sellable="$item->getSellable()"/>
+        @endforeach
```

Comment:
Another example of dynamic component use - this time for cart items.  If the sellable associated with the item has a custom component, it can be swapped in for the default here.